server/src/main/java/org/cloudfoundry/identity/uaa/scim/jdbc/JdbcScimUserProvisioning.java

```diff
@@ -239,19 +206,21 @@ public String[] mapFromInternal(final String[] attr) {
         }
 
         final String userFieldsWithPrefix = Arrays.stream(USER_FIELDS.split(","))
-                .map(field -> "u." + field)
+                .map(field -> joinPrefix + "." + field)
                 .collect(joining(", "));
         final String sql = String.format(
-                "select %s from users u join identity_provider idp on u.origin = idp.origin_key and u.identity_zone_id = idp.identity_zone_id where %s",
+                "select %s from users %s join identity_provider idp on %s.origin = idp.origin_key and %s.identity_zone_id = idp.identity_zone_id where %s",
```

Comment:
Not sure why this changed to parameterize the alias "u" for the users table, and also use a calculated value for it. Why the added complexity here?

Reply:
changed